Packages/Tracking/Core/Runtime/Scripts/XRHandsSubsystem/SubsystemStarter.cs

```diff
@@ -12,9 +13,20 @@ public class SubsystemStarter
         static XRHandProviderUtility.SubsystemUpdater updater = null;
 
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.AfterSceneLoad)]
-        private static void Awake()
+        private static void RunAfterSceneLoad()
         {
-            Application.quitting += OnDestroy;
+            Application.quitting += OnQuit;
+            LeapProvider leapProvider = Hands.Provider;
+            XRHandSubsystemProvider subsystemProvider = null;
+
+            if(leapProvider == null)
+            {
+                var providerGO = new GameObject();
+                var instantiated = GameObject.Instantiate(providerGO) as GameObject;
```

Comment:
I think you don't need to do this instantiate, as new GameObject() already creates one for you?